data/data/alibabacloud/cluster/vpc/vpc.tf

```
@@ -4,9 +4,17 @@ locals {
   prefix            = var.cluster_id
   newbits           = tonumber(split("/", var.vpc_cidr_block)[1]) < 16 ? 20 - tonumber(split("/", var.vpc_cidr_block)[1]) : 4
```

nit not related to this PR: This would be a lot easier to read if it used the `max` function.
```suggestion
  newbits           = max(4, 20 - tonumber(split("/", var.vpc_cidr_block)[1]))
```

---

Okey, thanks

---

# 5413

---

PR #5414